File: Python3/Excel Sheet Column Title.py
# 36ms, 100.0%
class Solution:
    def convertToTitle(self, n):
        res = ''
        while n:
            res = chr(ord('A') + (n - 1) % 26) + res    # chr: int-->char, ord: char-->int
            n = (n - 1) // 26
        return res
        

File: Java/Excel Sheet Column Title.java
// 0ms, 100.00%; 36.5MB, 5.13%
class Solution {
    public static String convertToTitle(int n) {
        StringBuilder sb = new StringBuilder();
        while (n > 0) {
            int x = (n - 1) % 26;
            sb.append((char) (x + 65));
            n = (n - x) / 26;
        }
        return sb.reverse().toString();
    }
}
